media-plugins/vdr-devstatus/vdr-devstatus-0.4.1-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit vdr-plugin-2
+
+DESCRIPTION="VDR plugin: display the usage status of the available DVB devices"
+HOMEPAGE="http://www.u32.de/vdr.html"
+SRC_URI="http://www.u32.de/download/${P}.tgz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
```

Comment:
You can't add these keywords, as you're not part of the dedicated arch teams. You need to ask keywording via Bugzilla, let me know if you need help doing that.

Reply:
learned that already... arch keywords have to be added through bugzilla...